Kitodo/src/main/java/org/kitodo/production/services/data/RulesetService.java

```diff
@@ -202,10 +190,8 @@ public Map<String, Object> findByTitleAndFile(String title, String file) throws
      * @return search result
      */
     public List<Map<String, Object>> findByTitleOrFile(String title, String file) throws DataException {
-        BoolQueryBuilder query = new BoolQueryBuilder();
-        query.should(createSimpleQuery(RulesetTypeField.TITLE.getKey(), title, true));
-        query.should(createSimpleQuery(RulesetTypeField.FILE.getKey(), file, true));
-        return findDocuments(query);
+        // TODO delete method stub
+        throw new UnsupportedOperationException("no longer used function");
```

Comment:
Method is not marked as deprecated but is throwing this exception? Why not remove this method and maybe adjust the still used usage places?